osu.Game/Screens/Edit/Components/Menus/EditorMenuBar.cs

```diff
@@ -14,19 +17,56 @@ namespace osu.Game.Screens.Edit.Components.Menus
 {
     public partial class EditorMenuBar : OsuMenu
     {
+        private const float heading_area = 114;
+
         public EditorMenuBar()
             : base(Direction.Horizontal, true)
         {
             RelativeSizeAxes = Axes.X;
 
             MaskingContainer.CornerRadius = 0;
-            ItemsContainer.Padding = new MarginPadding { Left = 100 };
+            ItemsContainer.Padding = new MarginPadding { Left = heading_area };
```

Comment:
The items container can scroll and will be behind the header. I would also not hardcode padding if we want to localise the header and would use a grid container with the header out of `EditorMenuBar`.

![osu!_FMcx59tLbb](https://github.com/ppy/osu/assets/35318437/cf54672d-edf6-4ead-8b21-1c8d9b88716e)

Reply:
Masking should be applied to the menu in this instance, good catch.

Reply:
Should be resolved.